src/App.js

```diff
@@ -1,5 +1,148 @@
+    this.#profit = ((this.getPrizeMoney() / this.#money) * 100).toFixed(1);
+    Console.print(`${MESSAGE.PROFIT_FRONT}${this.#profit}${MESSAGE.PROFIT_BACK}`);
+
+    this.exitLottery();
+  };
+
+  printLotteryResult = () => {
+    Console.print(LOTTERY_RESULT.TITLE);
+    Console.print(`${LOTTERY_RESULT.MATCHED_THREE}${this.#lotteryResult[PRIZE_GRADE.FIFTH]}${LOTTERY_RESULT.UNIT}`);
+    Console.print(`${LOTTERY_RESULT.MATCHED_FOUR}${this.#lotteryResult[PRIZE_GRADE.FOURTH]}${LOTTERY_RESULT.UNIT}`);
+    Console.print(`${LOTTERY_RESULT.MATCHED_FIVE}${this.#lotteryResult[PRIZE_GRADE.THIRD]}${LOTTERY_RESULT.UNIT}`);
+    Console.print(`${LOTTERY_RESULT.MATCHED_FIVE_AND_BONUS}${this.#lotteryResult[PRIZE_GRADE.SECOND]}${LOTTERY_RESULT.UNIT}`);
+    Console.print(`${LOTTERY_RESULT.MATCHED_SIX}${this.#lotteryResult[PRIZE_GRADE.FIRST]}${LOTTERY_RESULT.UNIT}`);
+
+    this.printProfit();
+  };
+
+  checkBonusNumber = (index) => {
+    if (this.#myLottery[index].includes(this.#bonusNumber)) return this.#lotteryResult[PRIZE_GRADE.SECOND]++;
+    else return this.#lotteryResult[PRIZE_GRADE.THIRD]++;
+  };
+
+  getLotteryResult = () => {
+    this.#matchedCounts.forEach((count, index) => {
+      if (count === 6) this.#lotteryResult[PRIZE_GRADE.FIRST]++;
+      if (count === 5) this.checkBonusNumber(index);
+      if (count === 4) this.#lotteryResult[PRIZE_GRADE.FOURTH]++;
+      if (count === 3) this.#lotteryResult[PRIZE_GRADE.FIFTH]++;
+    });
+
+    this.printLotteryResult();
+  };
+
+  compareLotteryNumbers = (lottery) => {
+    let matchedCount = 0;
+    lottery.forEach((number) => {
+      if (this.#winningNumbers.includes(number)) matchedCount++;
+    });
+    return matchedCount;
+  };
+
+  getMatchedCounts = () => {
+    this.#myLottery.forEach((lottery) => {
+      const numberOfMatchedNumbers = this.compareLotteryNumbers(lottery);
+      this.#matchedCounts.push(numberOfMatchedNumbers);
+    });
+    this.getLotteryResult();
+  };
+
+  checkIsValidBonusNumber = (bonusNumber, winningNumbers) => {
+    return new BonusNumber(bonusNumber, winningNumbers);
+  };
+
+  getBonusNumber = () => {
+    Console.readLine(MESSAGE.GET_BONUS_NUMBER, (bonusNumber) => {
+      this.#bonusNumber = Number(bonusNumber);
+      this.checkIsValidBonusNumber(this.#bonusNumber, this.#winningNumbers);
+      this.getMatchedCounts();
+    });
+  };
+
+  checkIsValidWinningNumber = (winningNumbers) => {
+    return new Lotto(winningNumbers);
+  };
+
+  getWinningNumber = () => {
+    Console.readLine(MESSAGE.GET_WINNING_NUMBERS, (winningNumbers) => {
+      this.#winningNumbers = winningNumbers.split(",").map(Number);
+      this.checkIsValidWinningNumber(this.#winningNumbers);
+      this.getBonusNumber();
+    });
+  };
+
+  getLotteryNumbers = () => {
+    const lotteryNumbers = [];
+    for (let i = 0; i < this.#numberOfLottery; i++) {
+      lotteryNumbers.push(Random.pickUniqueNumbersInRange(1, 45, 6).sort((a, b) => a - b));
+    }
+    return lotteryNumbers;
```

Comment:
```suggestion
    return [...Array(this.#numberOfLottery)].map(() => Random.pickUniqueNumbersInRange(1, 45, 6).sort((a, b) => a - b));
```
이렇게 간단하게 바꿀 수 있을 것 같아요!

Reply:
이 부분 바꿔보고 싶었는데 map을 사용할 생각은 못 했었네요. 감사해요!